modules/internal/ChapelBase.chpl

```diff
@@ -1131,6 +1125,7 @@ module ChapelBase {
     return ir;
   }
   
+  pragma "compiler generated" 
   pragma "donor fn"
   pragma "auto copy fn"
   inline proc chpl__autoCopy(x) return chpl__initCopy(x);
```

Comment:
Hm, this may break things if chpl__initCopy() is user defined. Right?
Do you know under what conditions this is going to comply with whatever requirements the "compiler generated" pragma promises? Maybe split into two versions, distinguished by where-clause. One version would have that pragma and the other wouldn't?